### DISCUSSION TITLE
Don't show my own repos

### DISCUSSION THREAD

**Discussion by sindresorhus** [community/community#13349] [Other Feature Feedback, Questions, & Ideas]
My own repo showed up in "For you" (I have starred the repo):

<img width="578" alt="Screen Shot 2022-03-23 at 23 04 48" src="https://user-images.githubusercontent.com/170270/159744966-36711edb-df29-42bd-821a-36c80d28361d.png">

You should also not show repos from orgs I'm part of.

**Comment by jfullstackdev** [comment]
it's a known issue, and Staff members are now aware of that

**Comment by AlinaFreed** [comment]
Hi
